runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -239,7 +237,8 @@ public static void populateFileDescriptorProtosInMessage(
     private void identifyMessageTypesinFileDescriptorProto(FileDescriptorProto fileDescriptorProto) {
         for (DescriptorProtos.DescriptorProto descriptorProto : fileDescriptorProto.getMessageTypeList()) {
             String messageName;
-            if (fileDescriptorProto.getPackage() == null || fileDescriptorProto.getPackage().equals("")) {
+            fileDescriptorProto.getPackage();
```

Comment:
Was this intended to be removed?

Reply:
IDEA's warning:
<img width="754" alt="image" src="https://github.com/CorfuDB/CorfuDB/assets/314292/444d6e3e-f204-4fa3-9a4a-14791beabfb4">

Reply:
It looks like the result always will be at least: ""